src/Symfony/Component/String/Enum/TruncateCut.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String\Enum;
```

Comment:
Do we need a _Enum_ namespace there ?  

<personal opinion  😅> 

1/ Enum is still "[soft-reserved](https://www.php.net/manual/en/reserved.other-reserved-words.php)" for now (note: other reserved keywords do allow usage in namespaces)

2/ I'm not sure there is a real interest to do it ... as (again, personal opinion) i don't think we would have created a "Constant" folder here for instance.

</personal opinions  😅>

Reply:
I moved it in  `Symfony\Component\String`